packages/dev/docs/src/docs.css

```diff
@@ -432,6 +432,19 @@ div:global(.sp-layout) {
   background: none;
 }
 
+div:global(.sandpack) {
+  display: none;
+}
+
+div:global(.example-actions) {
+  margin-bottom: -20px;
+  margin-right: -35px;
```

Comment:
i don't know if i should be horrified or impressed, this makes sure there is always space below the example for the buttons and doesn't remove it from the document flow

Reply:
Yeah, wasn't pretty but it is responsive now. The issue was that on large screens, we have a ton of padding on examples, so we would probably rather not add extra space for the actions. And for smaller screens, there isn't any padding, so we want to add space for the actions.